doc/nrf/templates/sample_README.rst

```diff
@@ -61,34 +85,128 @@ Connect PIN1 to PIN2, then cut the connection again.
 
 User interface*
 ***************
+.. note::
+   Add button and LED assignments here, plus other information related to the user interface.
```

Comment:
Thread CLI example mentions here a link to CLI command list. Maybe the note should reflect this and allow for some other options than buttons and LEDs?

Reply:
Added.